socorro/external/boto/crashstorage.py

```diff
@@ -54,20 +63,23 @@ def __init__(self, config, namespace="", quit_check_callback=None):
         )
         self.connection_source = config.resource_class(config)
 
+        # Function for calling a function retriably
+        self.retryable = retry(
```

Comment:
This creates a decorator that wraps functions in a retryable. We use it later in this file to wrap all the `do_xyz` function calls.